app/scripts/background.js

```diff
@@ -786,10 +786,7 @@ browser.runtime.onInstalled.addListener(({ reason }) => {
 });
 
 function setupSentryGetStateGlobal(store) {
-  if (!global.rootGlobals) {
-    global.rootGlobals = {};
-  }
-  global.rootGlobals.getSentryState = function () {
+  global.sentryHooks.getSentryState = function () {
```

Comment:
Our code in the background and UI is run in the "root compartment", which is a LavaMoat-protected locked down environment that has special privileges above what typical packages would have. Packages have restricted endowments/capabilities as determined by the policy file, but we automatically get access to all packages and globals in the root compartment. This includes the `sentryHooks` object we set earlier.

Reply:
:+1: